scripts/build_upload.sh

```diff
@@ -96,7 +96,7 @@ echo "Removing first bokeh build at $first_build_loc"
 #########################
 
 # build for each python version
-for py in 27 33 34;
```

Comment:
Thinking about people that might have to use 3.3 without a choice (organizational mandate, etc) 



> On Aug 21, 2015, at 19:36, Damián Avila <notifications@github.com> wrote:
> 
> In scripts/build_upload.sh:
> 
> > @@ -96,7 +96,7 @@ echo "Removing first bokeh build at $first_build_loc"
> >  #########################
> >  
> >  # build for each python version
> > -for py in 27 33 34;
> I don't think so... but if you want to use the 3.4 conda package you just need to make a 3.4 environment... I don't think it is too much work to ask...
> 
> —
> Reply to this email directly or view it on GitHub.
>